packages/libraries/federation-link-utils/README.md

```diff
@@ -0,0 +1,66 @@
+# GraphQL Hive - federation-link-utils
+
+[Hive](https://the-guild.dev/graphql/hive) is a fully open-source schema registry, analytics,
+metrics and gateway for [GraphQL federation](https://the-guild.dev/graphql/hive/federation) and
+other GraphQL APIs.
+
+---
+
+This library can be used to make a custom features for GraphQL schemas backed by Federation's
+[`@link`](https://www.apollographql.com/docs/graphos/reference/federation/directives#the-link-directive)
+directive.
+
+## Features
+
+- Link version support.
+- Import `as`/namespacing support that follows the [link spec](https://specs.apollo.dev/link/v1.0/).
+- Only `graphql` as a dependency.
+
+## Usage
+
+```graphql
+# schema.graphql
+
+directive @example(eg: String!) on FIELD
+extend schema @link(url: "https://specs.graphql-hive.com/example/v0.1", import: ["@example"])
+type Query {
+  user: User @example(eg: "query { user { id name } }")
+}
+
+type User {
+  id: ID!
+  name: String
+}
+```
+
+```typescript
+// specs.ts
```

Comment:
Let's add import statement here, that make it more obvious which functions/classes etc. are required for achieving the example below.

Reply:
Added